# Type: docs
# Subtype: D2
# English Summary: Modified the comments in the package-info.java file
# Chinese Summary: 修改 package-info.java 文件中的注释
# Entry Number: 8

# === Git Diff Content ===

diff --git a/ratpack-remote/src/main/java/ratpack/remote/package-info.java b/ratpack-remote/src/main/java/ratpack/remote/package-info.java
index 822ce1ad5..d9d0afb26 100644
--- a/ratpack-remote/src/main/java/ratpack/remote/package-info.java
+++ b/ratpack-remote/src/main/java/ratpack/remote/package-info.java
@@ -15,6 +15,6 @@
  */
 
 /**
- * Integration with <a href="http://groovy.codehaus.org/modules/remote/" target="_blank">Groovy Remote Control</a>.
+ * Integration with Groovy Remote Control.
  */
-package ratpack.remote;
\ No newline at end of file
+package ratpack.remote;
